eth2util/hash.go

```diff
@@ -40,3 +41,16 @@ func SlotHashRoot(slot eth2p0.Slot) ([32]byte, error) {
 
 	return hash, nil
 }
+
+// SHA256 calculates the sha256 checksum of the input data. https://github.com/ethereum/consensus-specs/blob/v0.9.3/specs/core/0_beacon-chain.md#hash
+func SHA256(data []byte) [32]byte {
```

Comment:
actually, this is a trivial function, please remove as anyone should just inline the use of sha256 if required.